.github/workflows/docker.yml

```diff
@@ -52,30 +55,43 @@ jobs:
 
       - name: Get version
         id: version
-        run: echo "::set-output name=RELEASE_VERSION::$(cat VERSION)"
+        run: echo "RELEASE_VERSION=$(cat VERSION)" >> $GITHUB_OUTPUT
 
       - name: Compute target tag
         id: target
         env:
           RELEASE_VERSION: ${{ steps.version.outputs.RELEASE_VERSION }}
         run: |
-          # Translate the Github reference into a tag name.
+          # Translate the GitHub reference into a tag name.
           #
-          # - `release` tag maps to `zeek:latest`
+          # - `latest` tag maps to `zeek:latest`
+          # - `lts` tag maps to `zeek:lts`
           # - `v*` tag (excluding `v*-dev` tags) maps to `zeek:RELEASE_VERSION`
           # - `master` branch maps to `zeek-dev:latest`
           #
           # Any other refs are not published below.
-          if [ "${GITHUB_REF}" = "refs/tags/release" ]; then
-            echo "::set-output name=tag::zeek:latest"
+          if [ "${GITHUB_REF}" = "refs/tags/latest" ]; then
+            echo "tag=zeek:latest" >> $GITHUB_OUTPUT
+          elif [ "${GITHUB_REF}" = "refs/tags/lts" ]; then
+            echo "tag=zeek:lts" >> $GITHUB_OUTPUT
           elif [ "${GITHUB_REF}" = "refs/heads/master" ]; then
-            echo "::set-output name=tag::zeek-dev:latest"
+            echo "tag=zeek-dev:latest" >> $GITHUB_OUTPUT
           elif [[ "${GITHUB_REF}" = refs/tags/v* ]] && [[ "${GITHUB_REF}" != refs/tags/v*-dev ]]; then
-            echo "::set-output name=tag::zeek:${RELEASE_VERSION}"
+            echo "tag=zeek:${RELEASE_VERSION}" >> $GITHUB_OUTPUT
           fi
 
-      - name: Login to DockerHub
+      - name: Login to ECR
+        # Don't publish on forks. Also note that secrets for the login are not
+        # available for pull requests, so trigger on pushes only.
+        if: github.repository == 'zeek/zeek' && github.event_name == 'push'
         uses: docker/login-action@v1
```

Comment:
Should this be `login-action@v2` as well?

Reply:
Indeed. Thank you for catching that. Fixed.